### PR TITLE
Social Anti-RightClick: codedosa.com

### PR DESCRIPTION
**Pull request by ghost**
### URL
https://codedosa.com/1602

### CODE REVIEW

File: EnglishFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -594,6 +594,8 @@ uaclips.com,kzclip.com,ru-clip.com,ruclip.com###morev
 !
 !
 !
+! 
+codedosa.com#$#body * { visibility: visible!important; }
```
Comment (zloyden) [inline EnglishFilter/sections/antiadblock.txt:598]: What is this rule for?

Comment (piquark6046) [reply EnglishFilter/sections/antiadblock.txt:598]: To prevent anti-adblock script.

Comment (piquark6046) [reply EnglishFilter/sections/antiadblock.txt:598]: When AG is on:
scre1
When AG is off:
scre2

Comment (zloyden) [reply EnglishFilter/sections/antiadblock.txt:598]: Not reproduced on my end. What's caused this? Please write steps to reproduce.

Comment (piquark6046) [reply EnglishFilter/sections/antiadblock.txt:598]: Filter List:
AdGuard Base, AdGuard Mobile Ads, AdGuard URL Tracking, AdGuard Tracking Protection, Fanboy's Anti-Facebook List, Fanboy's Anti-thirdparty Fonts, AdGuard Social Media, AdGuard Annoyances, Online Malicious URL Blocklist, Spam404, NoCoin Filter List, AdGuard DNS, AdGuard Turkish, AdGuard French, AdGuard Russian, AdGuard Chinese, AdGuard German, AdGuard Japanese, AdGuard Dutch, AdGuard Spanish/Portuguese

How to reproduce:
1. Just visit https://codedosa.com/1602
2. Then, you can see the blank page.

Comment (piquark6046) [reply EnglishFilter/sections/antiadblock.txt:598]: @zloyden

Comment (zloyden) [reply EnglishFilter/sections/antiadblock.txt:598]: Don't need to mention me, if you leave a comment. We see this in our email.
Btw, next time please provide a filter or rule that causes the problem.
Here it's caused by `Fanboy's Anti-thirdparty Fonts` -> `||googleapis.com/ajax/libs/webfont/$third-party`.
You don't have to fix it - I will report it to their author. Please remove the rule.

Comment (piquark6046) [reply EnglishFilter/sections/antiadblock.txt]: Okay.
https://github.com/AdguardTeam/AdguardFilters/pull/90429/commits/24c672b38a3cd573d1373a54d1954d31ed219417

Comment (zloyden) [reply EnglishFilter/sections/antiadblock.txt]: Well done.

File: EnglishFilter/sections/antiadblock.txt [hunk 2]
```diff
@@ -596,7 +596,7 @@ uaclips.com,kzclip.com,ru-clip.com,ruclip.com###morev
 !
 ! https://github.com/AdguardTeam/AdguardFilters/issues/90092
 @@||anonigviewer.com/assets/js/advertisements.js
-anonigviewer.com#%#//scriptlet("set-constant", "adblock", "false")
+anonigviewer.com#%#//scriptlet("set-constant", "adblock", "false"
```
Comment (zloyden) [inline EnglishFilter/sections/antiadblock.txt:599]: Oh, there is missed `)` at the end.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/antiadblock.txt (hunk 2 above)
Comment (piquark6046) [reply EnglishFilter/sections/antiadblock.txt]: Restored. https://github.com/AdguardTeam/AdguardFilters/pull/90429/commits/5f76190a52a569afec363b34a27adfa8767580fa

**Review by zloyden — APPROVED** [review]

**Pull request merged**